Description: Fix path to the GDM configuration file, which is different
 in Debian.
Author: Josselin Mouette <joss@debian.org>
Bug-Debian: http://bugs.debian.org/627311
Bug: https://bugs.freedesktop.org/show_bug.cgi?id=49993
---
 src/daemon.c |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- accountsservice.orig/src/daemon.c
+++ accountsservice/src/daemon.c
@@ -49,7 +49,7 @@
 #define PATH_SHADOW "/etc/shadow"
 #define PATH_NOLOGIN "/sbin/nologin"
 #define PATH_FALSE "/bin/false"
-#define PATH_GDM_CUSTOM "/etc/gdm/custom.conf"
+#define PATH_GDM_CUSTOM "/etc/gdm3/daemon.conf"
 
 #define USERDIR LOCALSTATEDIR "/lib/AccountsService/users"
 
@@ -445,7 +445,7 @@ reload_autologin_timeout (Daemon *daemon
         daemon->priv->autologin_id = 0;
 
         if (!load_autologin (daemon, &name, &enabled, &error)) {
-                g_debug ("failed to load gdms custom.conf: %s", error->message);
+                g_debug ("failed to load gdms daemon.conf: %s", error->message);
                 g_error_free (error);
                 g_free (name);
 
